components/mdns/include/mdns.h

```diff
@@ -544,7 +544,7 @@ esp_err_t mdns_query_async_delete(mdns_search_once_t* search);
  *      True if search has finished before or at timeout
  *      False if search timeout is over
  */
-bool mdns_query_async_get_results(mdns_search_once_t* search, uint32_t timeout, mdns_result_t ** results);
+bool mdns_query_async_get_results(mdns_search_once_t* search, uint32_t timeout, mdns_result_t ** results, uint8_t * num_results);
```

Comment:
This is an API update, but since this function has been recently added, we could break the API now in `v5.0`